notebooks/official/workbench/gaming_churn_prediction/churn_prediction_for_game_developers.ipynb

```diff
@@ -446,7 +354,7 @@
         "id": "S4qnPgig91P1"
       },
       "source": [
-        "If you are using ***Vertex AI Workbench managed notebooks instance***, every cell which starts with \"#@bigquery\" will be a SQL Query. If you are using Vertex AI Workbench user managed notebooks instance or Colab it will be a markdown cell."
+        "If you're using ***Vertex AI Workbench managed notebooks instance***, every cell which starts with \"#@bigquery\" is a SQL Query. If you're using a Vertex AI Workbench user-managed notebooks instance or Colab, it's a markdown cell."
```

Comment:
add "a":
- "If you're using a ***Vertex AI Workbench managed notebooks instance***, "